graph/pkg/identity/cs3.go

```diff
@@ -20,6 +20,10 @@ type CS3 struct {
 	Logger *log.Logger
 }
 
+func (i *CS3) CreateUser(ctx context.Context, user libregraph.User) (*libregraph.User, error) {
+	panic("not implemented!") // TODO do NOT under any circumstances let this garbage make it to production.
```

Comment:
🤣 the comment

Reply:
Yeah. I just found that as well. Will fix it an return a proper error. :man_facepalming: